src/cmake/modules/FindJPEGTurbo.cmake

```diff
@@ -29,7 +29,22 @@ endif ()
 if (JPEG_INCLUDE_DIR AND JPEG_LIBRARY)
   set (JPEG_INCLUDE_DIRS ${JPEG_INCLUDE_DIR} CACHE PATH "JPEG include dirs")
   set (JPEG_LIBRARIES ${JPEG_LIBRARY} CACHE STRING "JPEG libraries")
-  file(STRINGS "${JPEG_INCLUDE_DIR}/jconfig.h"
+
+  find_path(jconfig_header_file jconfig.h PATHS "${JPEG_INCLUDE_DIR}")
+
+  if (NOT jconfig_header_file)
+    file(GLOB_RECURSE jconfig_header_file_list "${JPEG_INCLUDE_DIR}/*/jconfig.h")
+
+    if (jconfig_header_file_list)
+      list(GET jconfig_header_file_list 0 jconfig_header_file)
+    endif()
+
+    if (NOT jconfig_header_file)
+      message(FATAL_ERROR "Cannot find jconfig.h from libturbojpeg")
```

Comment:
I'm not so sure about the FATAL_ERROR here -- if this header can't be found, I think we want only the find_package(JPEGTurbo) to not succeed, and then to fall back on regular find_package(JPEG). We don't want the entire cmake build to fail.

Reply:
@illwieckz Any comment or further action on this? In general, I think it's fine except for the FATAL_ERROR part.

Reply:
Sorry, I was busy and totally forgot about it. I'll look at it again.

Reply:
@illwieckz Ping -- can you please take a look here at my previous comment and offer an opinion?

Reply:
Changed to a `WARNING`.